webapp/i18n/en.json

```diff
@@ -286,7 +284,7 @@
   "admin.email.selfPush": "Manually enter Push Notification Service location",
   "admin.email.skipServerCertificateVerification.description": "When true, Mattermost will not verify the email server certificate.",
   "admin.email.skipServerCertificateVerification.title": "Skip Server Certificate Verification: ",
-  "admin.email.smtpPasswordDescription": " Obtain this credential from administrator setting up your email server.",
+  "admin.email.smtpPasswordDescription": "The password for the SMTP server username.",
```

Comment:
@hmhealey @moonmeister Let's go with `The password associated with the SMTP username.` as the SMTP password help text